res/values/strings.xml

```diff
@@ -439,7 +439,7 @@
     <string name="tab_links">Links</string>
     <string name="tab_map">Map</string>
     <string name="tab_gallery_empty_hint">Images and videos shared in this chat will appear here.</string>
-    <string name="tab_docs_empty_hint">Documents, music, and other files shared in this chat will appear here.</string>
+    <string name="tab_docs_empty_hint">Documents and other files shared in this chat will appear here.</string>
```

Comment:
If the tab already says document, I would shorten to:

```suggestion
    <string name="tab_docs_empty_hint">Files shared in this chat will appear here.</string>
```

Reply:
>If the tab already says 'documents' [...]

the tab will say "Files" :) - see https://github.com/deltachat/deltachat-desktop/issues/2829 , https://github.com/deltachat/deltachat-android/pull/2341 , https://support.delta.chat/t/tabs-for-links-and-files/2142 

so, mentioning some types of files that are not covered by other tabs on any system probably makes some sense.